superset-frontend/packages/superset-ui-core/test/connection/callApi/parseResponse.test.ts

```diff
@@ -137,6 +137,17 @@ describe('parseResponse()', () => {
     expect(responseRaw.bodyUsed).toBe(false);
   });
 
+  it('resolves to big number value if `parseMethod=json-bigint`', async () => {
+    const mockBigIntUrl = '/mock/get/bigInt';
+    const mockGetBigIntPayload = '{ "value": 9223372036854775807 }';
+    fetchMock.get(mockBigIntUrl, mockGetBigIntPayload);
+    const responseBigNumber = await parseResponse(
+      callApi({ url: mockBigIntUrl, method: 'GET' }),
+      'json-bigint',
+    );
+    expect(`${responseBigNumber.json.value}`).toEqual('9223372036854775807');
+  });
```

Comment:
Thanks for adding the test!

Reply:
Do we also mock a `-2^63`  to test int64? and we also need to add a comment to point out the integer type is that singed int64.

Reply:
@zhaoyongjie good point. I added the spec for negative number.